packages/asset-service/src/generateAssetData/osmosis/index.ts

```diff
@@ -17,33 +18,72 @@ type OsmoAsset = {
   name: string
   display: string
   symbol: string
+  traces: {
+    type: string
+    counterparty: {
+      chain_name: string
+      bse_denom: string
+      channel_id: string
+    }
+    chain: {
+      channel_id: string
+    }
+  }[]
   logo_URIs: {
     png?: string
     svg?: string
   }
   coingecko_id: string
-  pools?: {
-    [key: string]: number
-  }
+  keywords: string[]
 }
 
 type OsmosisAssetList = {
-  chain_id: string
+  chain_name: string
```

Comment:
I may be out of the loop but why this name change?

Reply:
@cjthompson Osmosis changed the schema upstream. See [here](https://github.com/osmosis-labs/assetlists)